Makefile.in

```diff
@@ -1222,6 +1227,8 @@ stamps/build-llvm-newlib: $(LLVM_SRCDIR) $(LLVM_SRC_GIT) $(BINUTILS_SRCDIR) $(BI
 	cp $(notdir $@)/lib/LLVMgold.$(SHARED_LIB_SUFFIX) $(INSTALL_DIR)/lib
 	cd $(INSTALL_DIR)/bin && ln -s -f clang $(NEWLIB_TUPLE)-clang && \
 	    ln -s -f clang++ $(NEWLIB_TUPLE)-clang++
+	cd $(INSTALL_DIR)/bin && echo "-march=${MARCH} -mabi=${MABI} -mtune=${MTUNE}" >> $(LINUX_TUPLE)-clang.cfg \
```

Comment:
Why change to the installation directory? I would use `echo "-march=${MARCH} -mabi=${MABI} -mtune=${MTUNE}" > "$(INSTALL_DIR)/bin/$(XYZ_TUPLE)-clang.cfg"`.